pkg/image/importer/importer.go

```diff
@@ -397,6 +418,7 @@ func importRepositoryFromDocker(ctx gocontext.Context, retriever RepositoryRetri
 			importDigest.Err = err
 			continue
 		}
+		importDigest.Err = calculateImageSize(ctx, repo, importDigest.Image)
```

Comment:
Schema v1 supported by hub.docker.com. Before that we used is not documented structure to calculate the size.

https://github.com/docker/distribution/blob/master/docs/spec/manifest-v2-1.md#manifest-field-descriptions

```
v1Compatibility string

V1Compatibility is the raw V1 compatibility information. This will contain the JSON
object describing the V1 of this image.

```